Realm/TestUtils/RLMTestCase.m

```diff
@@ -23,7 +23,7 @@
 #import <Realm/RLMSchema_Private.h>
 #import <Realm/RLMRealmConfiguration_Private.h>
 
-static NSString *parentProcessBundleIdentifier()
+static NSString *parentProcessBundleIdentifier(void)
```

Comment:
This is a non-C++ source file and `()` in C does bad deprecated things which now produce a warning.